### ISSUE TITLE
[Firefox] Ask.com Text-ads get parsed only after toggling cosmetic filters

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#919]

**Comment by cqx931** [comment]
The content do get rendered when I do inspect element, and I see corresponding rules triggered in the ublock logger. However this is what I got from the PARSER, pretty strange...
![image](https://cloud.githubusercontent.com/assets/3056154/23741445/751a241c-04e3-11e7-8f6c-51a942d4fcac.png)

**Comment by cqx931** [comment]
on hold for web-extension
